collectors/ebpf.plugin/README.md

```diff
@@ -149,6 +149,7 @@ accepts the following values: ​
     new charts for the return of these functions, such as errors. Monitoring function returns can help in debugging
     software, such as failing to close file descriptors or creating zombie processes.
 -   `update every`:  Number of seconds used for eBPF to send data for Netdata.   
+-   `pid table size`: Defines the maximum number of PIDs stored inside the application hash table.
```

Comment:
Could we explain how changing `pid table size` affects memory usage? For example, if I change `pid table size` to `65536`, does that potentially double memory usage? Or, if I feel like the plugin is using too much memory, can I reduce `pid table size` in half to halve the memory usage?

Reply:
There is this kind of relationship and we can define the minimum expected per cpu, for example, if each element of a table needs `X bytes`, the amount of allocated memory is equal the number of elements times `X bytes`. These are the current tables and default values that we are delivering:

| Table name | Size of each element | Number of entries | total allocated bytes |
|---------------|---------|--------|----------|
|tbl_pid_stats| 128 | 32768 | 4194304 | 
|cstat_pid | 32| 32768 | 4194304| 
|tbl_conn_ipv4 | 56 | 16384| 917504 |
|tbl_conn_ipv6 | 56 | 16384| 917504 |
|tbl_nv_udp_conn_stats| 8 | 4096 | 32768|

Reply:
I will add this table in our documentation.

Reply:
@joelhans I updated the documentation adding your comment https://github.com/netdata/netdata/issues/10949#issuecomment-819771770, please, take a look in the table I added here for we define the necessity to add it or not to the documentation.

Reply:
@thiagoftsm Thanks for adding it! We should remove the first two paragraphs in that section, however, the ones that read:

> Because eBPF monitoring is complex, we are evaluating the performance of this new collector in various real-world conditions, across various system loads, and when monitoring complex applications.
>
> Our initial testing shows the performance of the eBPF collector is nearly identical to our apps.plugin collector, despite collecting and displaying much more sophisticated metrics. You can now use the eBPF to gather deeper insights without affecting the performance of your complex applications at any load.

Reply:
All right, I removed them. Thanks!